Code/Editor/TrackView/TrackViewNodes.cpp

```diff
@@ -2580,10 +2582,11 @@ void CTrackViewNodesCtrl::Update()
             {
                 const CTrackViewAnimNode* track = static_cast<const CTrackViewAnimNode*>(node);
                 if (track)
-                { 
-                    record->setText(0, track->GetName());
+                {
+                    AZStd::string trackName = track->GetName();
```

Comment:
nit: These could be const references to avoid a copy in future if `GetName()` it updated to return an `AZStd::string` (even if not the const ref will extend the lifetime of the temporary created if necessary)

Reply:
(this comment also applies to the other diffs above)